client/src/components/streams/StreamList.js

```diff
@@ -123,11 +149,20 @@ class StreamList extends React.Component {
 
   render() {
     return (
-      <div>
-        <h2 className="white-text">Streams</h2>
-        <div class="carousel">Div for Carousel here</div>
-        <div className="ui grid container">{this.renderList()}</div>
-        {this.renderCreate()}
+      <div id="stream-show" className="ui grid stackable">
```

Comment:
I notice you also used these ids in StreamEdit. It works, but I think technically these should be reserved for the elements in StreamShow, since there's only supposed to be a single element with a specific id. If someone wanted to change the style of StreamShow by editing the #stream-show selector, they would also be modifying the style of the stream list and stream edit unknowingly.

But we're getting close to the cutoff, and this is just a class project, so I wouldn't sweat it, but I thought it was worth mentioning.

Reply:
Yeah, I was worried that might happen. Would it be better to create a duplicate class but with a different name?

Reply:
I think so. Though if it's too much of a hassle maybe don't worry about it, since as far as I can see it works.